### DISCUSSION TITLE
Zelda breath of the wild.

### DISCUSSION THREAD

**Discussion by Alkpote81** [skyline-emu/skyline#1741] [Game Requests]
Bonjour à tous, déjà un grand merci pour cet emulateur, 
Alors je joue sur un note 10+, exynos, 12 gb de ram, gpu mali g76 mp12, et le jeux ne fonctionne tout simplement pas. Il ne doit être pas compatible encore, mais bizarre des gens arrive à le faire tourner enfin il me semble, pas de dlc ni de mise à jour j'ai essayer avec la base du jeux. Rien à faire, j'aimerais savoir si il est compatible ou pas du tout merci d'avance.

**Comment by QuackingCanary** [comment]
Salut, tu as de la chance que je parle français pour te répondre. Le serveur étant international il vaut mieux utiliser l'anglais ! Il faut que tu regardes sur GitHub, il y a une liste de compatibilité et Zelda BOTW n'en fait pas parti. Skyline a affiche son premier jeu en début d'année et il va falloir lui laisser un peu de temps ;)
Et pour ce qui est des mises à jour et des DLC, ils deviendront compatibles plus tard.
En espérant avoir répondu à toutes tes questions !

PS : ton téléphone utilisant une carte graphique Mali, les résultats peuvent être altérés et/ou retardés

Adrien

**Comment by Alkpote81** [comment]
Super je te remercie beaucoup, je m'en douter un peu mais bon ça viendra
avec le temps, et OK c noter pour l'anglais 😁 et vivement qu'il mette les
driver pour les gpu mali.

**Reply by QuackingCanary** [reply]
Pour les drivers on ne les aura pas, ils sont faits pour les processeurs Snapdragon (utilisant Adreno comme GPU). Les drivers pour les GPU Mali sont déjà optimisés et il n'y en a pas d'autres (sauf si des développeurs s'en chargent plus tard)

**Comment by Alkpote81** [comment]
Je vois je vois, je viens de voir la liste des compatibilité en effet ça
aide beaucoup, je vais essayer darksider warmastered edition en espérant
qu'il se lance et merci de ton aide c'est gentil, j'aurais une dernière
question: est ce que mon appareil suffit pour jouer ou faut il que je passe
un Soc snapdragon ?

**Reply by QuackingCanary** [reply]
Oui ça permet de savoir quels jeux fonctionnent et ceux qui ne fonctionnent pas avant de tester. Si jamais le jeu n'a pas été testé, n'hésite pas à ouvrir une issue pour nous le faire savoir !
Tout dépend quels jeux tu veux faire tourner, un GPU Mali n'est pas optimal et le Note 10+ a déjà quelques années. Si tu as la possibilité d'avoir un téléphone utilisant un Snapdragon ça sera mieux pour l'émulation en général

**Comment by Alkpote81** [comment]
Oui pas de soucis j'essayerai de partager tout ça, j'essaye de faire
tourner des gros jeux quand même mais bon pour l'instant c'est un échec, je
vais essayer des petit jeux pour voir, et le note 10 commence à se faire
vieux c'est vrais mais bon je pense qu'avec ma ram (12gb)  je devrait
pouvoir faire tourner quelque jeux quand même

**Reply by QuackingCanary** [reply]
Pour l'instant aucun téléphone ne peut lancer convenablement de gros jeux, l'émulateur est trop récent !
L'avantage c'est que tu as 12go de ram, j'espère que tu pourras faire tourner les jeux que tu veux !!

**Comment by Alkpote81** [comment]
Oui c'est le seul avantage que j'ai je ne comptait pas trop sur mon Soc et
mon gpu, merci pour t'est réponse et de ton aide ! J'essayerais des petit
jeux en attendant, au plaisir l'ami ;)

**Reply by QuackingCanary** [reply]
Avec plaisir, n'hésite pas à rejoindre le serveur Discord ! ;)
(https://discord.gg/skyline-emu)

**Comment by ghost** [comment]
Bonjour

**Comment by Sandor60** [comment]
Bonjour j'aimerais de l'aide s'il vous plaît j'ai debuter sur l'émulateur Skyline et je n'arrive pas a faire tourner Pokémon lets go Pikachu a chaque fois sa plante pouvez vous m'aider et ou puige trouver la liste de chaque driver pour les autres jeux que je viens de télécharger s'il vous plaît aidé moi j'aimerais énormément jouer au jeux Nintendo Switch sur mon téléphone TCL 20 L je vous remercie si vous m'aider cordialement sandor 60
